pkg/handlers/upgrade_policies.go

```diff
@@ -0,0 +1,271 @@
+package handlers
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift/ocm-agent/pkg/consts"
+	log "github.com/sirupsen/logrus"
+)
+
+type UpgradePoliciesHandler struct {
+	ocm       *sdk.Connection
+	clusterId string
+}
```

Comment:
Most of the methods needs `upgradePolicyId string` too , can't it move it as a part of struct ?